yamlutils/yaml_test.go

```diff
@@ -12,11 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package swag
+package yamlutils
 
 import (
-	"encoding/json"
-	"net/http"
+	json "encoding/json"
```

Comment:
This alias is useless

```suggestion
	"encoding/json"
```

You can fix a few files

Reply:
yes I know. This is not really ready for a review at that level. Just a draft to experiment with new ideas re dependencies management with this repo. We'll resume the nitpicking chase in a bit :)

Reply:
done